databases/week2/db1w2hw_animalFarm_DDL_hwPart4.sql

```diff
@@ -0,0 +1,132 @@
+-- db2w1hw_animalFarmDB_ddl sumanghimire
+-- animalFarm DDL
+CREATE DATABASE animalFarm DEFAULT CHARACTER SET = 'utf8mb4';
+USE animalFarm;
+CREATE TABLE `owner` (
+  `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `firstName` VARCHAR(255) NOT NULL,
+  `lastName` VARCHAR(255) NOT NULL,
+  `DOB` DATE NULL,
+  `contact` VARCHAR(10) NOT NULL,
+  `address` VARCHAR(255) NOT NULL
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `farm` (
+  `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` VARCHAR(255) NOT NULL,
+  `address` VARCHAR(255) NOT NULL,
+  `description` VARCHAR(255) NOT NULL,
+  `phone` VARCHAR(10) NOT NULL
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `owner_farm` (
+  `owner_id` INT(10) unsigned NOT NULL,
+  `farm_id` INT(10) unsigned NOT NULL,
+  PRIMARY KEY(`owner_id`, `farm_id`),
+  CONSTRAINT `fk_owner_farm_owner` FOREIGN KEY (`owner_id`) REFERENCES `owner` (`id`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `fk_owner_farm_farm` FOREIGN KEY (`farm_id`) REFERENCES `farm` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `animal` (
+  `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `name` VARCHAR(255) NOT NULL,
+  `quantity` INT(255) NOT NULL,
+  `price` INT(255) NOT NULL,
+  `farm_id` INT(10) unsigned NOT NULL,
+  CONSTRAINT `fk_animal_farm` FOREIGN KEY (`farm_id`) REFERENCES `farm` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `customer` (
+  `id` INT(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  `firstName` VARCHAR(255) NOT NULL,
+  `lastName` VARCHAR(255) NOT NULL,
+  `email` VARCHAR(255) NULL,
+  `mobile` VARCHAR(10) NOT NULL,
+  `address` VARCHAR(255) NOT NULL
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+CREATE TABLE `sales` (
+  `quantity` VARCHAR(10) NOT NULL,
```

Comment:
what's the difference between "quantity" here and "quantity" in the animal table?